code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -295,15 +295,13 @@
 
 		if(T.density)
 			continue
-		for(var/obj/O in T)
```

Comment:
This code is what I think should have checked if the fire could pass a turf... But as you can see it was just a loop that did nothing at all.

Reply:
Does it really do nothing? It looks like its suppose to prevent fire spreading to turfs which have dense objects on them.

Reply:
It indeed looks like that. The issue it has is that it calls `continue` while it is in a `for` loop itself. Thus going to the next `O in T` instead of continuing the while loop
As a result this loop does nothing but continue if `O.density` is true. And continue if it is false

Reply:
Right, its a continue inside a for loop.. for some reason I was thinking it was going to continue the while. Yes this is useless.